lib/common.bash

```diff
@@ -174,3 +174,42 @@ clean_env_ctr()
 		sudo ctr c rm $(sudo ctr c list -q)
 	fi
 }
+
+# Restarts a systemd service while ensuring the start-limit-burst is set to 0.
+# Outputs warnings to stdio if something has gone wrong.
+#
+# Returns 0 on success, 1 otherwise
+restart_systemd_service_with_no_burst_limit() {
+	local service=$1
+	info "restart $service service"
+
+	local active=$(systemctl show "$service.service" -p ActiveState | cut -d'=' -f2)
+	[ "$active" == "active" ] || warn "Service $service is not active"
+
+	local start_burst=$(systemctl show "$service".service -p StartLimitBurst | cut -d'=' -f2)
+	if [ "$start_burst" -ne 0 ]
+	then
+		local unit_file=$(systemctl show "$service.service" -p FragmentPath | cut -d'=' -f2)
+		[ -f "$unit_file" ] || { warn "Can't find $service's unit file: $unit_file"; return 1; }
+
+		local start_burst_set=$(grep StartLimitBurst $unit_file | wc -l)
+		if [ "$start_burst_set" -eq 0 ]
+		then
+			sudo sed -i '/\[Service\]/a StartLimitBurst=0' "$unit_file"
```

Comment:
I saw curdini is used in the scripts, you could consider using it (I'm not sure if it's actually would be helpful)

Reply:
I prefer sed, but if  curdini is the preferred  tool for kata I'll change it, sure.